models/maespa/R/met2model.MAESPA.R

```diff
@@ -0,0 +1,265 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+#
+## R Code to convert NetCDF CF met files into MAESPA met files
+
+##If files already exist in "Outfolder", the default function is NOT to overwrite them and 
+##only gives user the notice that file already exists. If user wants to overwrite the existing files, just change 
+##overwrite statement below to TRUE.
+
+##' met2model wrapper for MAESPA
+##'
+##' @title met2model.MAESPA
+##' @export
+##' @param in.path location on disk where inputs are stored
+##' @param in.prefix prefix of input and output files
+##' @param outfolder location on disk where outputs will be stored
+##' @param start_date the start date of the data to be downloaded (will only use the year part of the date)
+##' @param end_date the end date of the data to be downloaded (will only use the year part of the date)
+##' @param overwrite should existing files be overwritten
+##' @param verbose should the function be very verbose
+##' 
+##'
+
+met2model.MAESPA <- function(in.path, in.prefix, outfolder, start_date, end_date, ..., overwrite=FALSE,verbose=FALSE){
+  
+  
+  library(PEcAn.utils)
+  print("START met2model.MAESPA")
+  start.date <- as.POSIXlt(start_date, tz = "GMT")
+  end.date<- as.POSIXlt(end_date, tz = "GMT")
+  out.file <- paste(in.prefix, strptime(start.date, "%Y-%m-%d"),strptime(end.date, "%Y-%m-%d"),"dat", sep=".")
+  out.file.full <- file.path(outfolder, out.file)
+  
+  results <- data.frame(file = out.file.full,
+                        host = fqdn(),
+                        mimetype ='text/csv',
+                        formatname = 'Sipnet.climna' ,
```

Comment:
I'm not sure what format MAESPA met is in, but I know it's NOT Sipnet.clim and I'm fairly sure it's not csv either. You need to update this to be the format name you registered with BETY or else it will break the met workflow and potentially both MAESPA and SIPNET runs (because PEcAn will let people run SIPNET with MAESPA formatted met).